functions-lab/exercise1/functions1.js

```diff
@@ -0,0 +1,8 @@
+var btn = document.querySelector('button');
+var text = document.querySelector('p');
+
+function swapText() {
+  text.innerHTML = 'Nice To Meet You';
+}
+
+btn.addEventListener('click', swapText);
```

Comment:
Exactly! The problem was having parenthesis at the end of swapText which "invoked" the function instead of "passing it" to "addEventHandler"

It's interesting to note that although JavaScript does not use variable types, many functions do expect specific types, so addEventListeners here expects a string and a callback function